src/MergeContext.js

```diff
@@ -368,8 +368,7 @@ class Labels
 //         immediately or after successful tests completion;
 // merged: with a staged commit that has been merged into the base branch.
 // Here, PR "staged commit" is a commit at the tip of the staging branch
-// pointed to by the PR tag. If a PR tag does not exist or does not point to
-// the tip of the staging branch, then the PR does not have a staged commit.
+// with commit message first line, having the PR number NNN (as "(#PRNNN)" string).
```

Comment:
If we require a leading space, then let's mention that here:

```suggestion
// with the commit title ending with the PR number (as " (#PRNNN)" string).
```

but it may be better to refer to the actual regex:


```suggestion
// with commit title ending with the PR number (see PrNumberRegex).
```

Reply:
Done.